content/docs/concepts/tsdb.md

```diff
@@ -0,0 +1,52 @@
+---
+title: TSDB
+sort_rank: 5
+---
+
+A times series is a series of **samples**. Each sample is a two-tuple consisting of a timestamp and a value:
+
+Example time series: the weather in Austin, TX.
+
+```
+[
+    (June 15th 2018 1:00 pm, 20.0 C),
+    (June 15th 2018 1:05 pm, 20.2 C),
+    (June 15th 2018 1:10 pm, 20.1 C),
+    ...
+]
+```
+
+Element | Precision | Data type
+:-------|:----------|:---------
+Timestamp | Millisecond | [int64](https://golang.org/pkg/builtin/#int64)
+Value | Depends on the use case | [float64](https://golang.org/pkg/builtin/#float64)
+
+## Storage problem
```

Comment:
This all seems like internals, so it doesn't belong in the "Concepts" section which is for new users.

This should be in simple docs in the tsdb repo, rather than on prometheus.io I think.

Reply:
As a higher level comment, I would like to see subcomponent docs being pulled into the main docs on p.io, most likely via includes pulled at build-time.

Deeper documentation can, and arguably should, be pulled into the main site; just clearly marked as such. This is beyond the scope of a reply to a comment on a line on a PR, though.